tests/monotouch-test/ARKit/ARSkeletonTest.cs

```diff
@@ -0,0 +1,24 @@
+﻿#if __IOS__
+
+using System;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using ARKit;
+using Foundation;
+using NUnit.Framework;
+using ObjCRuntime;
+
+namespace monotouchtest.ARKit {
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class ARSkeletonTest {
+		[Test]
+		public void UnknonwPointTest ()
+		{
+			using (var notKnownPoint = new NSString ("nariz"))
+				Assert.IsNull (ARSkeleton.CreateJointName (notKnownPoint));
```

Comment:
Can't add the success until visio is done, tracked here: https://github.com/xamarin/xamarin-macios/issues/9420